pkg/controllers/resources/idpbuilder.cnoe.io_custompackages.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.12.0
```

Comment:
looks like you need to bump the version of your controller gen. are you using an older version of idpbuilder or is there any other reason to have this older version used?